resources/public/api/conf/1.0/application.yaml

```diff
@@ -293,6 +304,7 @@ paths:
               schema:
                 oneOf:
                   - $ref: '#/components/schemas/badRequestParameters'
+                  - $ref: '#/components/schemas/badSubmissionGeneralDeclaration'
```

Comment:
Not sure I understand why we have both badRequestParameters & badSubmissionGeneralDeclaration here.

badSubmissionGeneralDeclaration seems to cover everything.

And on a slightly different note, badRequestParameters seems to only contain code and no message in the response schema